mercury/templates/dashboard.html

```diff
@@ -58,5 +59,46 @@ <h3>The Simulated Data for each data packet sent is shown:</h3>
         {#            Compression/Suspension of the 4 Wheels: FR: {{item.suspension_fr}}, FL: {{item.suspension_fr}}, BR: {{item.suspension_fr}}, BL: {{item.suspension_fr}}<br>#}
     </table>
 </div>
+
+	<div class="row">
+        <div id="articles" style="width:1400px;height:800px;"></div>
+    </div>
+	<script src="https://cdn.plot.ly/plotly-latest.min.js"></script>
+	<script src="http://code.jquery.com/jquery-3.3.1.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script>
```

Comment:
FYI: there is a jQuery in our repo so you don't have to use the one from the internet, if you want.

Reply:
Sure. For now, I will go with it. Will change it afterward